src/traits/impls.rs

```diff
@@ -18,7 +22,8 @@ use crate::{
 impl DoughnutApi for () {
     type PublicKey = [u8; 32];
     type Timestamp = u32;
-    type Signature = ();
+    type Signature = H512;
```

Comment:
Would prefer this was left as `()` or a really basic type like: `[u8 ;64]]`.
It's not version 0 specifc, so not `H512`